Casks/morpheus.rb

```diff
@@ -1,6 +1,9 @@
+# typed: false
+# frozen_string_literal: true
+
 cask "morpheus" do
```

Comment:
This shouldn't be here.
```suggestion
cask "morpheus" do
```

Reply:
I also thought so, just ran ' brew style --fix <cask>' as suggested by the PR comment.